pkg/kubelet/cm/devicemanager/plugin/v1beta1/server.go

```diff
@@ -53,6 +57,9 @@ type server struct {
 	rhandler   RegistrationHandler
 	chandler   ClientHandler
 	clients    map[string]Client
+
+	// Monitor whether the gRPC server has started
+	serverMonitor atomic.Bool
```

Comment:
we write it in a single goroutine. Why do we need an atomic here?

Reply:
Indeed, just a bool is sufficient.